pages.fr/common/ansible-playbook.md

```diff
@@ -0,0 +1,28 @@
+# ansible-playbook
+
+> Exécute les taches définies dans le playbook sur les machines distantes via SSH.
+> Plus d'informations : <https://docs.ansible.com/ansible/latest/cli/ansible-playbook.html>.
+
+- Exécute les taches d'un playbook :
+
+`ansible-playbook {{playbook}}`
```

Comment:
```suggestion
`ansible-playbook {{livre_de_jeu}}`
```

Google translation with spaces replaced with `_`.

Reply:
Hi Emily ! 
Well i did not translate `playbook` on purpose because `livre_de_jeu` is a literal translation and it's a false friend. `livre de jeu` means a book for games , like books who you are the hero or where's waldo etc. so we are a bit far from the playbook from ansible 😆 

Most of the time I look at official documentation translated in french or tutorials to know what is the translation used. But this time there is no translation 😓 .

So i that case i do not recommend to translate playbook ^^" 

what do you think ?

Reply:
I think we are fine to leave it as playbook.